airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -29,6 +29,8 @@ function configuredbt() {
     echo "Running: transform-config --config ${CONFIG_FILE} --integration-type ${INTEGRATION_TYPE} --out ${PROJECT_DIR}"
     # Generate a profiles.yml file for the selected destination/integration type
     transform-config --config "${CONFIG_FILE}" --integration-type "${INTEGRATION_TYPE}" --out "${PROJECT_DIR}"
+    # Remove config file as it might still contain sensitive credentials
+    rm "${CONFIG_FILE}"
```

Comment:
Normalization image is used by custom operations to translate a `destination_config.json` into a `profiles.yml` file.

We could make sure to remove the config file once we're done translating it

Reply:
I think I'm not totally clear on why this is needed since it's running a trusted image?

Reply:
> I think I'm not totally clear on why this is needed since it's running a trusted image? or is this re-used for custom operations?

Yes,the  normalization image is trusted to generate the `profiles.yml` (from a `destination_config.json` file). 
The resulting `profiles.yml` file is then transferred to run custom DBT operations in the user's specified docker image (so the latter one never needs to read/parse a `destination_config.json`)